R/tm_a_pca.R

```diff
@@ -13,20 +13,40 @@
 #' It controls the font size for plot titles, axis labels, and legends.
 #' - If vector of `length == 1` then the font sizes will have a fixed size.
 #' - while vector of `value`, `min`, and `max` allows dynamic adjustment.
-#' @templateVar ggnames "Elbow plot", "Circle plot", "Biplot", "Eigenvector plot"
-#' @template ggplot2_args_multi
+#' @param ggplot2_args `r roxygen_ggplot2_args_param("Elbow plot", "Circle plot", "Biplot", "Eigenvector plot")`
+#' @param decorators `r roxygen_decorators_param("tm_a_pca")`
```

Comment:
What is this sorcery : p?

Reply:
It's the recommended way for dealing with `roxygen2` tags.

It's all about having the `tm_<module_name>` as seen in image below. 

It's nice to have, but I'm more than happy to revert this and keep it simple as it was (ggplot2_args back to template and shared `@param decorators` with `"See "Decorating tm_<module_name> below"` or an equivalent generic text

![image](https://github.com/user-attachments/assets/a23057be-2c6f-4dae-bf02-6e08196d55f1)